habitat/sims/habitat_simulator.py

```diff
@@ -321,41 +316,9 @@ def action_space_shortest_path(
             the source. For the last item in the returned list the action
             will be None.
         """
-        assert agent_id == 0, "No support of multi agent in {} yet.".format(
-            self.__class__.__name__
+        raise NotImplementedError(
```

Comment:
What about leaving this implementation in and having two versions of shortest path:

1. Greedy shortest path which runs fast but not necessarily gives you the exact shortest path. 
2. Action space shortest path which gives you an accurate shortest path.

Reply:
See comment on `@pytest.mark.skip` for context.